knowledge-portal/source/README.md

```diff
@@ -0,0 +1,36 @@
+# Usage
+
+## Adding this pipeline to your repo
+
+You will have to modify the existing `gitlab-ci.yml` file to include:
+
+```yaml
+    stages:
+        - copy_to central_repo
+    include:
+        - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+
+```
+## Configure CI/CD variables
+
+1. Go to `Settings >> CI/CD >> Variables`
+2. Click on `Add variable`. 
+3. Give the name `PUSH_TOKEN` in `Key` field and `Value` field will be provided by Platform Team.
+4. Click on `Add variable` button to confirm changes. 
+
+## Creating `manifest.txt` file
+
+- The manifest file keeps track of README files you choose to import to `central-repo`.
+- It also allows renaming a file and import copies of same file to separate destination folders.   
```

Comment:
A slightly modified suggestion:

It also allows renaming a file and copying the same file to separate destination folders in case you would like to have different views of the same underlying data through force-duplication